functions/post_issue_message_test.ts

```diff
@@ -0,0 +1,44 @@
+import { SlackFunctionTester } from "deno-slack-sdk/mod.ts";
+import handler from "./post_issue_message.ts";
+import { stub } from "@std/testing/mock";
+import { assertEquals } from "@std/assert/equals";
+
+const { createContext } = SlackFunctionTester("my-function");
+
+Deno.test("No message found", async () => {
```

Comment:
```suggestion
Deno.test("Posts a message", async () => {
```

I'm not certain if we're also checking that a message is posted, but feel free to ignore this!